Working on getting portraits to show properly

kjdonahue opened the pull request.

Commit:
Working on getting portraits to show properly

Hydriad commented:
https://stackoverflow.com/questions/26648276/force-images-to-be-square-and-centered-in-css

Commit:
Adding photoset files

kjdonahue committed:
Update styles.css

This looks pretty nice, although it's still using two different styles for landscape vs portrait.

Hydriad reviewed (commented):
Commenting so this doesn't appear in my Review Requests tab anymore
